src/dsp_tools/utils/excel2json/resources.py

```diff
@@ -119,24 +120,9 @@ def _row2resource(
 
     # load the cardinalities of this resource
     try:
-        details_df = pd.read_excel(excelfile, sheet_name=name)
-    except ValueError:
-        # Pandas relies on openpyxl to parse XLSX files.
-        # A strange behaviour of openpyxl prevents pandas from opening files with some formatting properties
-        # (unclear which formatting properties exactly).
-        # Apparently, the excel2json test files have one of the unsupported formatting properties.
-        # The following two lines of code help out.
-        # Credits: https://stackoverflow.com/a/70537454/14414188
-        # pylint: disable-next=import-outside-toplevel
-        from unittest import mock
-
-        p = mock.patch("openpyxl.styles.fonts.Font.family.max", new=100)
-        p.start()
-        try:
-            details_df = pd.read_excel(excelfile, sheet_name=name)
-        except ValueError as err:
-            raise BaseError(str(err)) from None
-        p.stop()
+        details_df = utl.read_and_clean_excel_file(excelfile=excelfile, sheetname=name)
+    except ValueError as err:
+        raise BaseError(str(err)) from None
```

Comment:
I think this isn't necessary any more. All other usages of `read_and_clean_excel_file()` use it without try-catch.

I don't know why I had added this additional try-catch inside the first try-catch: If I look at it now, it doesn't make sense.

Reply:
This conversation is still unresolved. Do you agree, or would you like to leave the code as it is?

Reply:
Okay, that's fine. Thanks for the explanation, I didn't know that.